tests/functional/processes/ingest/test_doab_process.py

```diff
@@ -5,7 +5,6 @@
 from .assert_uploaded_manifests import assert_uploaded_manifests
 
 
-@pytest.mark.skipif(os.getenv('IS_CI') == 'true', reason="Skipping in CI environment")
```

Comment:
oop - let's remove the unused import as well!

Reply:
oops didn't see this, I'll create a separate PR for this.